index.html

```diff
@@ -1,18 +1,147 @@
-    })
-    .then(function (myText) {
-      console.log(myText);
-    });
-}
-
-</script>
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Yelpington</title>
+    <link rel= 'stylesheet' href="style.css"/>
+		<link
+			rel="stylesheet"
+			href="https://unpkg.com/leaflet@1.3.1/dist/leaflet.css"
+		/>
+    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/marked/marked.min.js"></script>
+	</head>
+
+	<body>
+    <div id = 'title'>Yelpington</div>
+		<div id="map"></div>
+		<header id="restaraunt">
+			<ul>
+				Restaraunt List
+				<li>
+					<a href="http://localhost:8080/restaurant.html#ahli-babas-kabob"
+						>Ahli Babas Kabob Shop</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:8080/restaurant.html#american-flatbread"
+						>American Flatbread</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:8080/restaurant.html#august-first"
+						>August First</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:8080/restaurant.html#city-market"
+						>City Market</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:8080/restaurant.html#el-cortijo"
+						>El Cortijo</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:8080/restaurant.html#farmhouse-grill"
+						>Farmhouse Grill</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:8080/restaurant.html#hen-of-the-wood"
+						>Hen of The Wood</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:8080/restaurant.html#honey-road"
+						>Honey Road</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:8080/restaurant.html#kountry-kart-deli"
+						>Kountry Kart Deli</a
+					>
+				</li>
+				<li>
+					<a href="http://localhost:8080/restaurant.html#mr-mikes">Mr. Mikes</a>
+				</li>
+				<li>
+					<a href="http://localhost:8080/restaurant.html#pascolo-ristorante"
+						>Pascolo Ristorante</a
+					>
+				</li>
+			</ul>
+		</header>
+
+		<script>
+			// 44.4759 73.2121
+
+			var map = L.map("map").setView([44.47629, -73.213], 15);
+
+			L.tileLayer("https://{s}.tile.openstreetmap.fr/hot/{z}/{x}/{y}.png", {
+				attribution:
+					'&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
+			}).addTo(map);
+
+			function getJson() {
+				fetch("/" + "all.json")
+					.then(response => {
+						return response.json();
+					})
+					.then(contents => {
+						console.log(contents);
+						contents.forEach(restaurant => {
+							fetch("/" + restaurant + ".json")
+								.then(response => {
+									return response.json();
+								})
+								.then(restaurantObject => {
+                  console.log(restaurantObject);
+									getGeoJson(restaurantObject);
+								});
+						});
+					});
+			}
+
+			function getGeoJson(location) {
```

Comment:
getGeoJson is a bit of a misnomer here since this function does more than just get the GeoJSON info from nominatim. I'd recommend calling it something along the lines setMarker, which is more descriptive of the end result of this function.